### ISSUE TITLE
Component Mocker Error - NpmModuleMongodb

### ISSUE THREAD

**Issue by samhatoum** [xolvio/meteor-jasmine#285]
I'm seeing this on a simple project on Meteor 1.2

```
=> Meteor server restarted
W20150927-14:42:29.356(-7)? (STDERR) Could not mock the export NpmModuleMongodb of the package npm-mongo. Will continue anyway. [TypeError: Cannot call method 'substring' of undefined] TypeError: Cannot call method 'substring' of undefined
W20150927-14:42:29.358(-7)? (STDERR)     at ObjectID.Object.defineProperty.get [as generationTime] (/Users/sam/.meteor/packages/npm-mongo/.1.4.39_1.1vl04fl++os+web.browser+web.cordova/npm/node_modules/mongodb/node_modules/bson/lib/bson/objectid.js:259:55)
W20150927-14:42:29.358(-7)? (STDERR)     at _getMetadata (/Users/sam/.meteor/packages/sanjo_jasmine/.0.16.2.zwed4h++os+web.browser+web.cordova/npm/node_modules/component-mocker/index.js:294:51)
W20150927-14:42:29.358(-7)? (STDERR)     at _getMetadata (/Users/sam/.meteor/packages/sanjo_jasmine/.0.16.2.zwed4h++os+web.browser+web.cordova/npm/node_modules/component-mocker/index.js:301:25)
W20150927-14:42:29.359(-7)? (STDERR)     at _getMetadata (/Users/sam/.meteor/packages/sanjo_jasmine/.0.16.2.zwed4h++os+web.browser+web.cordova/npm/node_modules/component-mocker/index.js:294:29)
W20150927-14:42:29.359(-7)? (STDERR)     at _getMetadata (/Users/sam/.meteor/packages/sanjo_jasmine/.0.16.2.zwed4h++os+web.browser+web.cordova/npm/node_modules/component-mocker/index.js:294:29)
W20150927-14:42:29.359(-7)? (STDERR)     at Object.mocker.getMetadata (/Users/sam/.meteor/packages/sanjo_jasmine/.0.16.2.zwed4h++os+web.browser+web.cordova/npm/node_modules/component-mocker/index.js:403:22)
W20150927-14:42:29.359(-7)? (STDERR)     at meteor://💻app/packages/sanjo_jasmine/packages/sanjo_jasmine.js:1478:1
W20150927-14:42:29.359(-7)? (STDERR)     at Function._.each._.forEach (meteor://💻app/packages/underscore/packages/underscore.js:142:1)
W20150927-14:42:29.359(-7)? (STDERR)     at meteor://💻app/packages/sanjo_jasmine/packages/sanjo_jasmine.js:1475:1
W20150927-14:42:29.359(-7)? (STDERR)     at Function._.each._.forEach (meteor://💻app/packages/underscore/packages/underscore.js:142:1)
W20150927-14:42:29.359(-7)? (STDERR)     at Object.MockGenerator.generateMocks (meteor://💻app/packages/sanjo_jasmine/packages/sanjo_jasmine.js:1471:1)
W20150927-14:42:29.360(-7)? (STDERR)     at Object.generateMocks (meteor://💻app/packages/underscore/packages/underscore.js:779:1)
W20150927-14:42:29.360(-7)? (STDERR)     at [object Object].executeSpecsUnitMode [as runTests] (meteor://💻app/packages/sanjo_jasmine/packages/sanjo_jasmine.js:1616:1)
W20150927-14:42:29.360(-7)? (STDERR)     at runWithEnvironment (meteor://💻app/packages/meteor/packages/meteor.js:1065:1)
W20150927-14:42:29.395(-7)? (STDERR) connections property is deprecated. Use getConnections() method
```

Any pointers?

**Comment by Sanjo** [comment]
Do you use the latest sanjo:jasmine 0.19.x? Maybe just remove the server/unit folder.

**Comment by Sanjo** [comment]
This is already fixes in 0.19.x.

**Comment by davidthornton** [comment]
That is to say: the issue has already been fixed in `sang:jasmine` version 0.19.x. At the time of writing, `sanjo:jasmine` is at version 0.21.0, which addresses this issue.
